app/text_buffer.py

```diff
@@ -1289,6 +1289,17 @@ def doSelectionMode(self, mode):
           mode - self.selectionMode)))
       self.redo()
 
+  def selectCurrentLine(self):
+    """
+      This function is used by the CTRL + L shortcut.
```

Comment:
Let's keep these text_buffer.py function comments more generic. Please remove this line about CTRL + L. I.e. keep the bit about what the function does, without referring to how it is triggered. 

The reason is that vi-like bindings or emacs-like bindings may use different bindings.